app/scripts/metamask-controller.js

```diff
@@ -416,6 +416,7 @@ export default class MetamaskController extends EventEmitter {
       ),
       tokenListController: this.tokenListController,
       provider: this.provider,
+      networkConfigurations: this.networkController.state.networkConfigurations,
```

Comment:
@DDDDDanica, I'm unsure, but don't we need the same logic in `mmi-controller.js`?

Reply:
Hey @albertolive i'm not 100% sure about mmi-controller relation with mm-controller.  In this new added feature, we pass in all networks to specify the preferences for each network for selected account. do you think we would need such feature in MMI as well?

Reply:
Ok, I think it's not needed for now. We'll focus on MetaMerge and then, if we need to do something with this, we'll let you know

Reply:
thank you !!